### PR TITLE
fix(workflow): Display relevant options in issue actions dropdown

### PR DESCRIPTION
**Pull request by mikellykels**
Previously, we would display all options (Add Bookmark, Remove Bookmark, Set as Unresolved, Delete) when you select an issue. This PR fixes this to show relevant dropdown options depending on the status/bookmark of the issue selected.

[FIXES WOR-1632](https://getsentry.atlassian.net/browse/WOR-1632)

# Before
Screen Shot 2022-03-08 at 11 35 01 PM

# After

**Add Bookmark**
Screen Shot 2022-03-08 at 7 08 52 PM

**Remove Bookmark**
Screen Shot 2022-03-08 at 7 08 11 PM

**Mark Unresolved**
Screen Shot 2022-03-08 at 11 33 56 PM

### CODE REVIEW

**Comment by scttcper** [comment]
I think this needs to show all options if they've selected beyond the current page. The "select all 83 issues" 
image

File: static/app/views/issueList/actions/actionSet.tsx [hunk 1]
```diff
@@ -214,7 +241,7 @@ function ActionSet({
         </ActionLink>
       )}
       <DropdownMenuControlV2
-        items={menuItems}
+        items={updatedMenuItems[0]}
```
Comment (scttcper) [inline static/app/views/issueList/actions/actionSet.tsx:244]: feels weird to make an array of menu items and then use the first one?

**Comment by scttcper** [comment]
I think the logic might need to look a bit more like `canMarkReviewed` that checks if the button should be enabled/disabled

**Comment by mikellykels** [comment]
I missed that there was a `hidden` prop in `MenuItemsProps` - updated to use `hidden` instead and follow the `canMarkReviewed` logic

**Review by scttcper — APPROVED** [review]

**Pull request merged**